### ISSUE TITLE
[Browse] Show context menu from title

### ISSUE THREAD

**Issue by VWoeltjen** [nasa/openmctweb#33]
Show context menu for an object when user clicks (regular left-click) on the down arrow next to a domain object's title in Browse mode.

**Comment by VWoeltjen** [comment]
Would also like this as a reusable component which can also be shown next to labels in a layout.

**Comment by slhale** [comment]
We talked about in the future there possibly being different context menus for Browse and Edit mode. Does this mean that right now I should make the menu arrow's context menu available in both Browse and Edit mode, or just Browse mode?

**Comment by slhale** [comment]
I have a problem that I think requires some design decisions to be fixed, so I can't really fix it myself. 

The ContextMenuAction was previously set to dismiss the context menu on a click. I had to change this, as it would register the initial left click on the menu arrow as this click to close the context menu, and so never show the context menu. I changed it to mousedown, which allowed the context menu to be shown by either normal contextmenu actions, or left clicking the menu arrow. But, I just realized that this does not allow the user to actually click on anything inside the context menu. Because a mousedown is registered before a click, the mousedown to close the context menu happens before the click to do the action inside the context menu. 

What should I do to fix the behavior? 
(Or is there another way that I don't know about?)
